src/main/java/com/codicesoftware/plugins/hudson/actions/CheckoutAction.java

```diff
@@ -80,6 +83,10 @@ private static String removeNewLinesFromSelector(String selector) {
         return selector.trim().replace("\r\n", "").replace("\n", "").replace("\r", "");
     }
 
+    private static boolean SelectorNeedsUpdate(String selector) {
```

Comment:
I would rename `SelectorNeedsUpdate(...)` to `isChangesetOrLabelSelector(...)`

Reply:
You're right, the casing was incorrect... I'm so used to C# that I didn't even notice it 😯 I used the name you suggested and changed the condition so it makes sense 😉